mailpoet/tests/integration/Segments/WooCommerceTest.php

```
@@ -684,6 +684,47 @@ public function testGuestCustomerUnsubscribedWhenSignupConfirmationEnabled(): vo
     verify($subscriber->getStatus())->equals(SubscriberEntity::STATUS_UNSUBSCRIBED);
   }
 
+  public function testItSynchronizesCustomersWithMixedNullAndZeroCustomerIds(): void {
+    global $wpdb;
+
+    $guest1 = $this->insertGuestCustomer();
+    $guest2 = $this->insertGuestCustomer();
+    $guest3 = $this->insertGuestCustomer();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$guest3'. (undefined)

(UnusedLocalVariable)